src/filesystems/qsys/QSysFs.ts

```diff
@@ -131,40 +129,23 @@ export class QSysFS implements vscode.FileSystemProvider {
     async getMemberAttributes(connection: IBMi, path: QsysPath & { member?: string }) {
         const loadAttributes = async () => await connection.content.getAttributes(path, "CREATE_TIME", "MODIFY_TIME", "DATA_SIZE");
```

Comment:
Use not deprecated `getContent()`.
```suggestion
        const loadAttributes = async () => await connection.getContent().getAttributes(path, "CREATE_TIME", "MODIFY_TIME", "DATA_SIZE");
```